sbtplugin/src/main/scala/com/typesafe/tools/mima/plugin/SbtMima.scala

```diff
@@ -46,7 +46,9 @@ object SbtMima {
   /** Reports binary compatibility errors.
    *  @param failOnProblem if true, fails the build on binary compatibility errors.
    */
-  def reportErrors(problemsInFiles: Map[ModuleID, (List[core.Problem], List[core.Problem])],
+  def reportErrors(module: ModuleID,
```

Comment:
It took me a while to figure out what could potentially be using this method, that would be affected by this ABI change. Is it only sbt plugins that depends on sbt-mima and calls this method?

Anyway, added a new method instead of changing the signature of the old one.